tests.py

```diff
@@ -100,3 +107,21 @@ def test_sip_api_qtpy():
         assert sip.getapi("QString") == 2, ("PyQt4 API version should be 2, "
                                             "instead is %s"
                                             % sip.getapi("QString"))
+
+
+def test_sip_api_already_set():
+    """Qt.py should cause ImportError when sip API v1 was already set
+    (Python 2.x only)"""
+
+    if sys.version_info[0] == 2:
+        # Python 2.x
+        with pyqt4():
+            def import_qt():
+                import Qt
+
+            from PyQt4 import QtCore
+            import sip
+            sip.setapi("QString", 1)
+            assert_raises(ImportError, import_qt)
+    else:
+        raise SkipTest("Skipped because Python version is not 2.x")
```

Comment:
Hmmmm, this itches somewhat.

How about this?

```python
if py == 3:
  def test_sip_api_already_set():
```

Where py is defined as..

```python
py = sys.version_info[0]
```

That way, tests stay focused on a single version.